Files/MultilingualResources/Files.fr-FR.xlf

```diff
@@ -3256,21 +3256,22 @@
         </trans-unit>
         <trans-unit id="SettingsPrivacyPolicyMarkdownBlock.Text" translate="yes" xml:space="preserve">
           <source>
-*Personal Information Collection* 
+*Personal Information Collection*
 Files does not collect, store, share or publish any personal information.
 
 *Non-personal Information Collection*
 
 We use App Center to keep track of app usage, find bugs, and fix crashes. All information sent to App Center is anonymous and free of any user or contextual data.
   </source>
-          <target state="translated">
-*Collecte d'informations personnelles* 
+          <target state="needs-review-translation">
```

Comment:
Is this state intentional?

You can use this state to let others (or yourself in the future) know that this string should be reviewed in the future to check if it can be improved, but if you think this is already properly translated, it should be "translated".

Reply:
It was not intentional. I believe this was changed automatically in Visual Studio because the string changed. I have a plugin that automatically removes spaces from end of lines and "Multilingual App Toolkit Extension" may have changed the status. I have not changed this text. I will test tomorrow. I changed the status in a new commit.